dev-docs/dev.md

```diff
@@ -365,14 +335,14 @@ speed up the build.
     ```
 
     The output of the `dir` command should list the following 3 exe files in the
-    `C:\Users\${USER}\ops-agent\tmp\out\bin` directory:
+    `C:\Users\{{USER}}\ops-agent\tmp\out\bin` directory:
 
     *   `fluent-bit.exe`
     *   `google-cloud-metrics-agent_windows_amd64.exe`
     *   `google-cloud-ops-agent.exe`
 
     <details>
-    <summary>Sample output:</summary>
+    <summary>Sample output</summary>
 
     ```
         Directory: C:\Users\{{USERNAME}}\tmp\out
```

Comment:
nit: Since you remove most of the usage of {{USERNAME}}, i suggest also changing this output to match it.

Reply:
Nice catch! I have replaced 5 places from {{USERNAME}} to $env:UserProfile so that they are consistent with the rest of the doc.